src/select/select.js

```diff
@@ -334,11 +336,17 @@ class Select extends React.Component<PropsT, StatelessStateT> {
 
   getDropDown() {
     const {overrides, type, rows, selectedOptions} = this.props;
+    let maxHeight = this.props.maxHeight;
+    if (__BROWSER__ && parseInt(maxHeight) > window.innerHeight) {
+      // only for pixel-sized maxHeight
+      maxHeight = '80vh';
+    }
     const options = this.getOptions();
     const {isDropDownOpen, optionsLoaded} = this.state;
     const dropDownProps = {
       rows,
```

Comment:
We should remove `rows` is it's now used.

Reply:
removed